sources/driverdog/src/main.rs

```diff
@@ -282,7 +288,17 @@ fn load_modules_sets(
     target: Option<String>,
 ) -> Result<()> {
     // Update the modules.dep before we attempt to load kernel modules
-    let args: Vec<String> = Vec::new();
+    // We need to confirm if there is a specific modules-set and if so, add a preference for that into depmod
+    let mut args: Vec<String> = Vec::new();
+    if modules_sets.contains_key("nvidia-tesla")
+        && Path::new("/usr/share/nvidia/tesla-depmod.conf").exists()
+    {
+        args.push("--config=/usr/share/nvidia/tesla-depmod.conf".to_string());
+    } else if modules_sets.contains_key("nvidia-open-gpu")
+        && Path::new("/usr/share/nvidia/open-gpu-depmod.conf").exists()
+    {
+        args.push("--config=/usr/share/nvidia/open-gpu-depmod.conf".to_string());
+    }
```

Comment:
`driverdog` is a general program to link / load modules and this behavior is too specific to NVIDIA drivers. If you need a depmod.conf file then it should be in the TOML config file rather than hardcoded.

Reply:
I removed this since I was able to provide the functionality we need without this additional logic.